plugins/db-postgres/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@amplication/plugin-db-postgres",
-  "version": "1.2.1",
+  "version": "1.2.2-beta-2",
```

Comment:
are we using beta only for testing?

Reply:
yep, once it's approved I'll publish a stable version